compiler/z/codegen/InstOpCode.hpp

```diff
@@ -42,6 +42,7 @@ class InstOpCode: public OMR::InstOpCodeConnector
       OMR::InstOpCodeConnector(m) { }
 
    /* OMR TODO: Delegating method; need to be removed and replace with getMnemonic */
+   const char* getMnemonicName()                                        { return TR::InstOpCode::metadata[_mnemonic].name; }
    TR::InstOpCode::Mnemonic getOpCodeValue()                            { return _mnemonic; }
    TR::InstOpCode::Mnemonic setOpCodeValue(TR::InstOpCode::Mnemonic op) { return _mnemonic = op; }
```

Comment:
Was this TODO meant for the two functions below your change? If so we should move it down.

Reply:
27171ed